python/tests/test_logistic_regression.py

```diff
@@ -329,6 +329,88 @@ def test_params(tmp_path: str, caplog: LogCaptureFixture) -> None:
     _test_input_setter_getter(LogisticRegression)
 
 
+@pytest.mark.parametrize(
+    "Estimator,input_spark_params,cuml_params_update",
+    [
+        (
+            LogisticRegression,
+            {"regParam": 0.1, "elasticNetParam": 0.5},
+            {"penalty": "elasticnet", "C": 10.0, "l1_ratio": 0.5},
+        ),
+        (
+            LogisticRegression,
+            {"maxIter": 13},
+            {"max_iter": 13},
+        ),
+        (
+            LogisticRegression,
+            {"regParam": 0.25, "elasticNetParam": 0.0},
+            {"penalty": "l2", "C": 4.0, "l1_ratio": 0.0},
+        ),
+        (
+            LogisticRegression,
+            {"regParam": 0.2, "elasticNetParam": 1.0},
+            {"penalty": "l1", "C": 5.0, "l1_ratio": 1.0},
+        ),
+        (
+            LogisticRegression,
+            {"tol": 1e-3},
+            {"tol": 1e-3},
+        ),
+        (
+            LogisticRegression,
+            {"fitIntercept": False},
+            {"fit_intercept": False},
+        ),
+        (
+            LogisticRegression,
+            {"standardization": False},
+            {"standardization": False},
+        ),
+        (
+            LogisticRegression,
+            {"enable_sparse_data_optim": True},
+            None,
+        ),
+        (
+            LogisticRegression,
+            {"verbose": True},
+            {"verbose": True},
+        ),
+    ],
+)
+def test_copy(
```

Comment:
this is used in other tests so seems better to move in a generic central place like utils?